udata/templates/macros/issues.html

```diff
@@ -1,8 +1,8 @@
 {% macro issues_btn(obj, icons=None) %}
-{% set basename = obj.__class__.__name__|lower %}
 <button type="button" class="btn btn-danger btn-issues"
     data-api-url="{{ url_for('api.issues', for=obj.id|string) }}"
-    data-subject="{{ obj.id }}"
+    data-subject-id="{{ obj.id }}"
+    data-subject-class="{{ obj.__class__.__name__ }}"
```

Comment:
But here, to be able to do that properly, the property needs to be declared on each model because right now we don't have a common ancestor.
I agree with you, template should stay simple, but in this specific case, the main point is that it's a standard Python object (a class instance) instead of a specific inherited `*Discussion`.